components/feedbackBar.js

```diff
@@ -92,7 +100,7 @@ export class FeedbackBar extends Component {
     this.setState({ commentSubmitted: true });
     let payload = {
       whatWereYouDoing: this.state.action,
-      whatWentWrong: this.state.failure,
+      howCanInfoBeMoreUseful: this.state.failure,
```

Comment:
I think this should go to the column whatWentWrong in airtable. Maybe if you renamed the state variables (`action` and `failure`) it would be more obvious what's going on.